proposals/0724-qualified-lists.rst

```diff
@@ -0,0 +1,395 @@
+Qualified Lists
+===============
+
+.. author:: Brandon Chinn
+.. date-accepted:: Leave blank. This will be filled in when the proposal is accepted.
+.. ticket-url:: Leave blank. This will eventually be filled with the
+                ticket URL which will track the progress of the
+                implementation of the feature.
+.. implemented:: Leave blank. This will be filled in with the first GHC version which
+                 implements the described feature.
+.. highlight:: haskell
+.. header:: This proposal is `discussed at this pull request <https://github.com/ghc-proposals/ghc-proposals/pull/724>`_.
+.. sectnum::
+.. contents::
+
+This proposal proposes extending ``-XQualifiedDo`` to literal lists, to enable more ergonomic and more powerful syntax than ``OverloadedLists``. Another way to view this proposal would be extending ``-XRebindableSyntax`` to literal lists, but only within a local scope.
+
+See also:
+
+* `QualifiedStrings <https://github.com/ghc-proposals/ghc-proposals/pull/723>`_
+* `QualifiedNumerics <https://github.com/ghc-proposals/ghc-proposals/pull/725>`_
+
+Motivation
+----------
+
+Problems with Type Class-driven overloading
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+``OverloadedLists`` works by desugaring to a type-class-overloaded function. That leads to certain general shortcomings:
+
+* It is a module-wide setting
+
+  * Anecdotally, people would rather avoid ``OverloadedLists`` than deal with overloaded lists/strings in the entire module.
```

Comment:
since the proposals were split i think this doesn't need to reference strings anymore